src/BasePlatform.ts

```diff
@@ -139,6 +139,13 @@ export default abstract class BasePlatform {
         return false;
     }
 
+    /**
+     * Returns true if platform allows overriding native context menus
+     */
+    public allowOverridingNativeContextMenus(): boolean {
+        return true; // TODO: Change to false before merge! True for live preview
```

Comment:
**This needs to be changed before merging!**

Reply:
🚨🚨🚨

Reply:
@SimonBrandner does this still need changing? If so, can we make that change for merging?